recipes/spyder-vim/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "spyder-vim" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-vim-{{ version }}.tar.gz
+  sha256: 97c2ef31ae9bb24af1853d3741081cc27867f4c170e031a07daa213cc79577ca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - qtpy
+    - qtawesome
+    - spyder >=5.3.3
+
+test:
+  imports:
+    - spyder_vim
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/spyder-ide/spyder-vim
+  summary: A plugin to enable vim keybindings to the spyder editor
```

```suggestion
  summary: A plugin to enable Vim keybindings in the Spyder IDE's editor
```

Fix grammar, terminology and capitalization

---

I think this needs to also be changed on the plugin repo